yaml_files/simphony_metadata.yml

```diff
@@ -444,38 +443,34 @@ CUDS_KEYS:
   ######################################################
   # Conditions on Boundaries
   ######################################################
-  DIRICHLET:
-    definition: Dirichlet boundary condition, specify the value the solutions takes on the boundary of the domain
+  FIELD_CONDITION:
+    definition: Condition on (field) variables at domain boundaries encountered in the solution of partial differential equations (Direchlet, Neumann, Robin, etc)
     parent: CUBA.CONDITION
     models: [CUBA.CONTINUUM]
     CUBA.VARIABLE:
-      shape: (:)
-      default: []
+      shape: [1]
+    CUBA.VALUE:
```

Comment:
Considering that it's a very generic information at this level, wouldn't it make sense to have an H_VALUE on the ROBIN instead, and have H_VALUE defined as a shape 3 float array?